src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderCreator.java

```diff
@@ -73,7 +73,6 @@
     private final CaseDetailsConverter caseDetailsConverter;
     private final DocAssemblyService docAssemblyService;
     private final GenerateOrderRule generateOrderRule;
-    private final boolean jddoEnabled;
```

Comment:
Please remove the configurations from terraforms as well if this is not needed at all.

Reply:
Addressed